docs/quickstart.md

```diff
@@ -0,0 +1,203 @@
+# Quick Start Guide
+
+## Overview
+
+**Hypha** is an open-source tool for democratizing large-scale machine learning across heterogeneous compute resources. It helps discover and utilize diverse compute resources (CPUs, GPUs, and specialized accelerators) to run distributed training and inference with minimal operational overhead.
+
+This guide walks through installation, configuration, and running a first distributed training job.
+
+---
+
+## Prerequisites
+
+Make sure you have the following installed:
+
+| Requirement      | Version | Check Command     | Install Documentation                                                                                              |
+| ---------------- | ------- | ----------------- | ------------------------------------------------------------------------------------------------------------------ |
+| uv               | ≥ 0.9.7 | `uv --version`    | [https://docs.astral.sh/uv/getting-started/installation/](https://docs.astral.sh/uv/getting-started/installation/) |
+| git              | any     | `git --version`   | https://git-scm.com/book/en/v2/Getting-Started-Installing-Git                                                      |
+| rust (cargo)     | any     | `cargo --version` | https://www.rust-lang.org/tools/install                                                                            |
+| curl             | any     | `curl --version`  | https://curl.se/docs/install.html                                                                                  |
+| openssl / libssl | any     | `openssl version` | https://www.openssl.org/source/                                                                                    |
```

Comment:
Only `uv` is needed if one's not building from Source. `curl` is needed for the installer.